test/data/gtoc/correction-invoice.json

```diff
@@ -0,0 +1,142 @@
+{
+	"$schema": "https://gobl.org/draft-0/envelope",
+	"head": {
+		"uuid": "8a51fd30-2a27-11ee-be56-0242ac120002",
+		"dig": {
+			"alg": "sha256",
+			"val": "e85d130d30c22e788532169d683d88a33de290a1beaf6733f61b1b5cd647cd75"
+		}
+	},
+	"doc": {
+		"$schema": "https://gobl.org/draft-0/bill/invoice",
+		"$regime": "DE",
+		"$addons": [
+			"en16931"
```

Comment:
This should be `eu-en16931-v2017`, and will need to be regenerated to ensure all there.